.travis.yml

```diff
@@ -41,19 +34,19 @@ before_install:
   - pip install --no-cache-dir -U -r requirements-test.txt
   - ./runcheckmigration
   - flake8
-  - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then ./runisort; fi
+  - ./runisort
   - npm install -g jslint
 
 # command to install requirements
 install:
   - pip install $DJANGO
+  - pip install django-model-utils --upgrade
   - python setup.py -q develop
 
 # command to run tests, e.g. python setup.py test
 script:
   - jslint django_x509/static/django-x509/js/*.js
   - coverage run --source=django_x509 runtests.py
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.4 ]]; then ./tests/manage.py makemigrations --dry-run | grep "No changes detected"; fi
```

Comment:
why are you removing the check? We should drop the conditional but keep the check:

```
./tests/manage.py makemigrations --dry-run | grep "No changes detected";
```